onnxruntime/core/providers/webnn/builders/model_builder.cc

```diff
@@ -215,6 +215,8 @@ Status ModelBuilder::RegisterModelInputOutput(const NodeArg& node_arg, bool is_i
     ORT_RETURN_IF_NOT(SetWebnnDataType(desc, data_type), "Unsupported data type");
   }
 
+  emscripten::val::module_property("reserveTensor")(name, desc, is_input);
```

Comment:
What does this change for? I didn't find `reserveTensor` definition from `ts` files.

Reply:
Oops, I accidentally included code from my performance enhancement branch.